### ISSUE TITLE
about the visualization

### ISSUE THREAD

**Issue by yanqian123** [yysijie/st-gcn#96]
@yysijie hello, there is another question that i want to ask. I notice that on the video about "Attention+Prediction" there is a label "voting result : (doing something)" and around the skeleton there is another label ,sometimes the two labels are the same, and sometime there are different. I guess if every skeleton's json file has a action label, and these labels will devote to the final result ? and how to make a influence? or these labels just relate to the only one json file?? I don't know , could you give me some suggestion

**Comment by yysijie** [comment]
Hi, @yanqian123,
Actually, our model for prediction is based on Convolutional Neural Networks.
According to the properties of convolution, our model always predicts the result of a section of the full sequence which selected by a sliding window. As this sliding window moving, the model output of a series of results. These are the changing results you observed. The length of a section and the moving step of the sliding window is trivial and usually determined by the network structure.
After that, we average all results, and get the "voting result".

In short, the changing label related to a section of the sequence which centers on the current instant.

**Comment by yanqian123** [comment]
thank you a lot
